llvm/test/CodeGen/PowerPC/v2i64_scalar_to_vector_shuffle.ll

```diff
@@ -170,42 +170,42 @@ define <2 x i64> @test_v16i8_none(i8 %arg1, ptr nocapture noundef readonly %b) {
 ; CHECK-LE-P8-NEXT:    lxvd2x vs0, 0, r4
 ; CHECK-LE-P8-NEXT:    xxswapd v2, vs0
 ; CHECK-LE-P8-NEXT:    mtfprd f0, r3
-; CHECK-LE-P8-NEXT:    xxmrgld v2, v2, vs0
+; CHECK-LE-P8-NEXT:    xxpermdi v2, v2, vs0, 2
```

Comment:
it should be `xxpermdi v2, v2, vs0, 0` ?

Reply:
I think this should be resolved now from our offline discussion, but there was an issue in the previous codegen where we produced a `SCALAR_TO_VECTOR_PERMUTED` but we did not fix up the shuffle mask. This patch updates the shuffle mask, so this codegen should be correct.